src/pages/instances/InstanceSnapshots.tsx

```diff
@@ -29,6 +33,29 @@ const InstanceSnapshots: FC<Props> = ({ instance }) => {
   const [selectedNames, setSelectedNames] = useState<string[]>([]);
   const [processingNames, setProcessingNames] = useState<string[]>([]);
 
+  const onSuccess = (message: ReactNode) => {
+    setInTabNotification(success(message));
+  };
+
+  const onFailure = (title: string, e: unknown) => {
+    setInTabNotification(failure(title, e));
+  };
+
+  const {
+    data: project,
```

Comment:
We should feed the project as a prop from the parent, that already loaded it.

Reply:
It doesn't seem to me that the parent (`InstanceDetail`) has loaded the project config.